PSCoreScripts/New-ApimApi.ps1

```diff
@@ -0,0 +1,87 @@
+###########################################################################################
```

Comment:
We don't need this header - it's in the PSCoreScripts folder and uses the AZ cmdlets

Reply:
Removed headers from both files